src/Services/UserService.php

```diff
@@ -138,12 +136,9 @@ public function getUpdatePageParams(RequestInterface $request)
      * @return bool
      * @throws \Exception
      */
-    public function updateEntity($request)
+    public function updateEntity(string $name, string $role, string $id)
```

Comment:
return type and `updateEntity` is bad naming. you update a user

Reply:
also, isn't it a natural for the method to have the `id` parameter first? I feel it's a bit common sense